src/constants.ts

```diff
@@ -10,4 +10,19 @@ export const defaultEndpoints: Required<Endpoints> = {
   universalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/answers/autocomplete',
   verticalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/answers/vertical/autocomplete',
   filterSearch: 'https://liveapi-cached.yext.com/v2/accounts/me/answers/filtersearch',
-};
\ No newline at end of file
+} as const;
```

Comment:
for my own understanding, why is the const assertion needed?

Reply:
this marks all the properties as read only

Reply:
it also keeps the type as narrow as possible, though that doesn't matter here
here's a good post on it https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/